presto-hive/src/test/java/io/prestosql/plugin/hive/AbstractTestHive.java

```diff
@@ -791,6 +798,18 @@ protected final void setup(String databaseName, HiveConfig hiveConfig, HiveMetas
                 new GenericHiveRecordCursorProvider(hdfsEnvironment, hiveConfig));
     }
 
+    protected HdfsConfiguration createTestHdfsConfiguration()
+    {
+        return new HiveHdfsConfiguration(
+                new HdfsConfigurationInitializer(
+                        new HdfsConfig(),
+                        ImmutableSet.of(
+                                new PrestoS3ConfigurationInitializer(new HiveS3Config()),
+                                new GoogleGcsConfigurationInitializer(new HiveGcsConfig()),
+                                new PrestoAzureConfigurationInitializer(new HiveAzureConfig()))),
```

Comment:
Since you also update `io.prestosql.plugin.hive.HiveTestUtils#createTestHdfsEnvironment` (IMO: good), why do you need `createTestHdfsConfiguration` here? Isn't `HDFS_ENVIRONMENT` const good enough?

(also, HDFS_ENVIRONMENT is used elsewhere in the class, so we better consistently use one hdfs env everywhere)